src/app/category-chart/trendline/category-chart-with-trendline.component.ts

```diff
@@ -0,0 +1,124 @@
+import { AfterViewInit, Component, ViewChild } from "@angular/core";
+import { IgxCategoryChartComponent } from "igniteui-angular-charts/ES5/igx-category-chart-component";
+
+@Component({
+    selector: "app-category-chart-with-trendline",
+    styleUrls: ["./category-chart-with-trendline.component.scss"],
+    templateUrl: "./category-chart-with-trendline.component.html"
+})
+export class CategoryChartWithTrendlineComponent implements AfterViewInit {
+
+    public data: any;
+    public includedProperties: any;
+
+    @ViewChild("chart")
+    public chart: IgxCategoryChartComponent;
+
+    constructor() {
+
+        const medals: any = [
+            {
+                ChinaBronze: 100,
+                ChinaGold: 100,
+                ChinaMedals: 110,
+                ChinaSilver: 100,
```

Comment:
@agoldenbaum all data items should have total Medals that equal to sum of Gold, Silver, and Bronze medals. However that is not the case for any of data items in this array. Can you correct values of Gold, Silver, and Bronze medals?